eigsep_motor_control/encoder.py

```diff
@@ -27,70 +27,76 @@ def get_encoder(self, motor):
 class Potentiometer:
 
     NBITS = 16  # ADC number of bits
-
-    # serial connection constants (BAUDRATE defined in main.py)
-    PORT = "/dev/ttyACM0"
+    PORT = "/dev/ttyACM0"  # serial connection constants
 
     def __init__(self):
-        """
-        Class for reading voltages from the potentiometers.
-        """
-        self.ser = serial.Serial(port=self.PORT, baudrate=BAUDRATE)
-        self.ser.reset_input_buffer()
+        while True:
```

Comment:
could you add a max number of tries and a counter so that we don't loop forever?